### ISSUE TITLE
What should I do to reproduce the performance of the paper

### ISSUE THREAD

**Issue by emjay73** [VisualComputingInstitute/TrackR-CNN#59]
What should I do to reproduce the performance of the paper by training the model on my local PC?
Would you mind if I ask you to share the configuration file for that?
And if it's possible, do I have to start training from the checkpoint file named  'converted' which I assume to be the weight file pre-trained on COCO and Mapillary?

**Comment by emjay73** [comment]
I trained this code starting from the 'converted' model using the given configuration file and got similar evaluation results as given 'conv3d_sep2' model which shows similar performance as paper on the validation set.  However, in the paper, epochs were set to 40 and a given configuration says the epochs are 5. I'm confused.

**Comment by pvoigtlaender** [comment]
I think it depends on the definition of an epoch which we at some point changed. Since you have the configuration which reproduces the results, stick to that.
@ahnonay can you further elaborate on that?

**Comment by ahnonay** [comment]
Hey, 
the trackrcnn_init.zip contains the COCO+Mapillary pretrained model. The conv3d_sep2 config contains the settings to reproduce our main model, which you can also download (the conv3d_sep2-00000005.zip).
Now, regarding the 5 vs 40 epochs: I agree that this is confusing. The "5" in the config file corresponds to the "40" in the paper. Recall that we always set our batch size to 8. For one epoch in the code, all possible batches of 8 consecutive frames are processed. Essentially this means that each video is processed 8 times during one epoch. When writing the paper, we therefore found it more intuitive to speak of 40 epochs (since in total, for our training, the videos are seen 40 times). This change was not propagated back into the code. Again, apologies for the confusion.

**Comment by emjay73** [comment]
I think I get it. Because of the sequence overlap, each frame was used multiple times as much as the sequence length, therefore, you multiplied sequence length to the epoch. Am I right? Thank you for your explanation.

**Comment by ahnonay** [comment]
Yes, that is correct

**Comment by emjay73** [comment]
Thanks! Closing the issue.
